### PR TITLE
py: fix order by ID

### PR DESCRIPTION
**Pull request by isinyaaa**
Fixes: https://github.com/kubeflow/model-registry/issues/353

## Description
Updates the orderBy.Id -> ID as that's what's expected on MLMD.

## Merge criteria:

- All the commits have been [_signed-off_](https://github.com/kubeflow/community/tree/master/dco-signoff-hook#signing-off-commits)  (To pass the `DCO` check)

- [ ] The commits have meaningful messages; the author will squash them after approval or in case of manual merges will ask to merge with squash.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work.
- [ ] Code changes follow the [kubeflow contribution guidelines](https://www.kubeflow.org/docs/about/contributing/).

If you have UI changes

- [ ] The developer has added tests or explained why testing cannot be added.
- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Verify that UI/UX changes conform the UX guidelines for Kubeflow.

### CODE REVIEW

**Comment by isinyaaa** [comment]
cc: @tarilabs we could also ensure that every orderBy param gets uppercased on the go server, wdyt?

File: api/openapi/model-registry.yaml [hunk 1]
```diff
@@ -1661,7 +1661,7 @@ components:
       explode: true
       examples:
         orderBy:
-          value: Id
+          value: ID
```
Comment (tarilabs) [inline api/openapi/model-registry.yaml:1664]: I'm not sure I follow this modification,
allegedly the current value was consistent with the enumeration here:

https://github.com/kubeflow/model-registry/blob/327e865819c865b9849ab89bb53730156e90675d/api/openapi/model-registry.yaml#L1351

**Review by tarilabs — COMMENTED** [review]
thank you @isinyaaa I have one question to start with please, below

File: api/openapi/model-registry.yaml (hunk 1 above)
Comment (isinyaaa) [reply api/openapi/model-registry.yaml:1664]: completely missed it! thanks. updated now.

**Review by tarilabs — APPROVED** [review]
thanks a lot @isinyaaa also for fixing the other issues,

/lgtm
/approve

**Pull request merged**